en/flight_controller/pixhack_v3.md

```diff
@@ -0,0 +1,54 @@
+# Pixhack V3
+
+The CUAV *Pixhack V3* flight controller board is a flexible autopilot intended primarily for manufacturers of commercial systems.
+
+The board is a variant of the SOLO Pixhawk 2 (PH2) flight controller, which is in turn based on the [Pixhawk-project](https://pixhawk.org/) **FMUv3** open hardware design. It runs PX4 on the [NuttX](http://nuttx.org) OS, and is fully compatible with both PX4 or ArduPilot (APM) firmware.
+
+*Pixhack V3* has significant improvements with respect to the original design, including better interface layout and the addition of vibration damping and a thermostat system
```

Comment:
There's a missing point at the end.

Reply:
This is a complete flight controller
The lead connectors are in front and behind